### PR TITLE
renderer: Add tracy support to all commands

### PR DESCRIPTION
**Pull request by EXtremeExploit**
had to add a new define specifically for commands, hope bookmists approves it and macdu the commands part

### CODE REVIEW

File: vita3k/renderer/CMakeLists.txt [hunk 1]
```diff
@@ -42,7 +42,7 @@ add_library(
 )
 
 target_include_directories(renderer PUBLIC include)
-target_link_libraries(renderer PUBLIC crypto display dlmalloc mem stb shader glutil threads config util vkutil)
+target_link_libraries(renderer PUBLIC crypto display dlmalloc mem modules stb shader glutil threads config util vkutil)
```
Comment (Macdu) [inline vita3k/renderer/CMakeLists.txt:45]: Sorry, but we really shouldn't have to make renderer dependant on modules. Maybe put the tracy files somewhere else like util ?

Comment (EXtremeExploit) [reply vita3k/renderer/CMakeLists.txt]: done

**Comment by bookmist** [comment]
I think, separate tracy.h in separate projects are better. It look like there is tracy_command don't use anything from modules/tracy.h

File: vita3k/util/include/util/tracy.h [hunk 2]
```diff
@@ -110,8 +123,12 @@ inline std::string to_debug_str(const MemState &mem) {
 #else
 #define TRACY_FUNC(name, ...) __TRACY_FUNC(tracy_module_name, name, ##__VA_ARGS__, , , , , , , , )
 #define TRACY_FUNC_M(module_name_var, name, ...) __TRACY_FUNC(module_name_var, name, ##__VA_ARGS__, , , , , , , , )
+#define TRACY_FUNC_COMMANDS(name) __TRACY_FUNC_COMMANDS(name)
+#define TRACY_FUNC_COMMANDS_SET_STATE(name) __TRACY_FUNC_COMMANDS_SET_STATE(name)
```
Comment (bookmist) [inline vita3k/util/include/util/tracy.h:127]: There is no __VA_ARGS__ in TRACY_FUNC_COMMANDS, so workaround for __VA_ARGS__ is not needed (and also TRACY_FUNC_COMMANDS... should be defined in both branches of #if)

Comment (EXtremeExploit) [reply vita3k/util/include/util/tracy.h:129]: does last push fix it?

Comment (bookmist) [reply vita3k/util/include/util/tracy.h:129]: yes.

**Review by bookmist — APPROVED** [review]

**Pull request merged**